day_6/exercises/burrito.js

```diff
@@ -16,11 +16,28 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(newTopping) {
+    this.toppings.push(newTopping);
+  }
+
+  removeTopping() {
+    this.toppings.pop();
+  }
+
+  changeProtein(newProtein) {
+    this.protein = [];
+    this.protein.push(newProtein);
+    dinner.protein = this.protein.toString();
+  }
+
 };
 
 var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
```

Comment:
It looks like `protein` is defined as "Beans" when `dinner` gets created in the constructor method. Reassigning protein to an array and then pushing the newProtein, only to pull out a new string is over-complicating things.

What's another way of reassigning `protein` that has a data type of a string?